src/NuGet.Services.Metadata.Catalog.Monitoring/Validation/Test/PackageValidatorContext.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Collections.Generic;
+using Newtonsoft.Json;
+using NuGet.Services.Metadata.Catalog.Helpers;
+
+namespace NuGet.Services.Metadata.Catalog.Monitoring
+{
+    public class PackageValidatorContext
```

Comment:
This seems like a lot of state to be going into a queue message. What if queue messages are lost? How do we recover?

Reply:
Once a message is in the queue, I think the chances of the message disappearing are probably roughly similar to that of a storage blob disappearing, which is not a situation we plan for right now. Please correct me if I'm wrong however.

Regarding failures of the message being pushed into the queue, the job pushing that message will retry again.